zadania/Program.cs

```diff
@@ -103,6 +103,61 @@ void summary(List<Tuple<int,int,int,int>> links)
     foreach (KeyValuePair<string,int> kvp in summary)
         Console.WriteLine("Podróż {0} odbyła się {1} razy", kvp.Key, kvp.Value);
 }
+int GetAllTo(int i, int j, List<Tuple<int,int,int,int>> links)
+{
+    IEnumerable<Tuple<int,int,int,int>> appropiateLinksList =
+                    from link in links
+                    where link.Item3 == i && link.Item4==j
+                    select link;
+    return appropiateLinksList.Count();
+}
+int GetAllFrom(int i, int j, List<Tuple<int, int, int, int>> links)
+{
+    IEnumerable<Tuple<int, int, int, int>> appropiateLinksList =
+                    from link in links
+                    where link.Item1 == i && link.Item2 == j
+                    select link;
+    return appropiateLinksList.Count();
+}
+void maxTo(List<Tuple<int,int,int,int>> links)
+{
+    int[] result = { 0, 0, 0 };
+    for(int i = 0; i<height; i++)
+    {
+        for(int j = 0; j<width; j++)
```

Comment:
Skąd bierzesz height i width? Funkcja nie powinna odwoływać się do obiektów spoza swojego zakresu, no chyba że jest metodą. W tym wypadku tak nie jest i nie powinno się brać tego spoza funkcji, bo nie wiadomo czym to jest